adapters/appnexus/appnexus.go

```diff
@@ -345,21 +347,40 @@ func (a *AppNexusAdapter) MakeRequests(request *openrtb.BidRequest) ([]*adapters
 		}
 	}
 
-	reqJSON, err := json.Marshal(request)
-	if err != nil {
-		errs = append(errs, err)
-		return nil, errs
-	}
+	imps := request.Imp
+	resArr := make([]*adapters.RequestData, 0)
```

Comment:
Could create resArr with an initial capacity of `(len(imps)+maxImpsPerReq-1)/maxImpsPerReq` so it allocates memory for all needed requests up front.

Reply:
Done!